### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by KennyCopp**
### Description

I created a markdown file for the SQL ROW_NUMBER() window function as part of the portfolio project in the "Learn Git & GitHub" course

### Type of Change

- Adding a new entry

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/window-functions/terms/row_number/row-number.md
Comment (KTom101) [inline content/sql/concepts/window-functions/terms/row_number/row-number.md:101]: Can we have a section here showing the results in a table like the others?


Chris|Blake|Manchester
Emily|Pugh|Manchester
Eric|Gustav|Louisville
Joseph|Chang|Raleigh
Mary|Greene|Raleigh

File: content/sql/concepts/window-functions/terms/row_number/row-number.md
Comment (KTom101) [inline content/sql/concepts/window-functions/terms/row_number/row-number.md:1]: Rename the folder to match the file name "row-number" (replace the underscore `_` with a dash `-`)

**Review by KTom101 — CHANGES_REQUESTED** [review]
@KennyCopp I've added some changes to this entry as part of the 1st review. Please let me know if there are any questions, Thanks.

File: content/sql/concepts/window-functions/terms/row_number/row-number.md
Comment (KennyCopp) [reply content/sql/concepts/window-functions/terms/row_number/row-number.md:1]: Thank you for your suggestions, I've updated the folder name and made the changes to row-number.md itself. I believe I've added the commit to the right place? Please let me know if there is anything else.

File: content/sql/concepts/window-functions/terms/row-number/row-number.md
Comment (KTom101) [inline content/sql/concepts/window-functions/terms/row-number/row-number.md:29]:
```suggestion

* `PARTITION BY` is an optional parameter used to create partitions (groups of rows). If it is not used, `ROW_NUMBER()` will treat the result set as one partition. It is important to note that `ROW_NUMBER()` is applied to each partition separately; so the first row in a partition is assigned '1', the second row is assigned '2', and so on.
```

**Review by KTom101 — COMMENTED** [review]
@KennyCopp Just a few more spots that need spacing, but otherwise good.

**Review by KTom101 — APPROVED** [review]
@KennyCopp I added the remaining line breaks/spacing here: [f767225](https://github.com/Codecademy/docs/pull/2000/commits/f767225888fa0b38b87bf05eb951a1ccae7a0b42). Please wait for a second review from another maintainer/collaborator.

**Review by SSwiniarski — APPROVED** [review]
@KennyCopp Looks good as is.

**Comment by SSwiniarski** [comment]
@caupolicandiaz were you still going to review this or can we merge?

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz were you still going to review this or can we merge?

merge

**Comment by SSwiniarski** [comment]
@KennyCopp Congrats on the new entry. Live link here:https://www.codecademy.com/resources/docs/sql/window-functions/row-number

image

**Pull request merged**
